Orange/widgets/visualize/owscatterplotgraph.py

```diff
@@ -1174,36 +1174,46 @@ def help_event(self, event):
         if self.scatterplot_item is None:
             return False
 
+        domain = self.data.domain
+        PARTS = (("Class", "Classes", 4, domain.class_vars),
+                 ("Meta", "Metas", 4, domain.metas),
+                 ("Feature", "Features", 10, domain.attributes))
+
+        def format_val(var, point_data, bold=False):
+            text = escape('{} = {}'.format(var.name, point_data[var]))
+            if bold:
+                text = "<b>{}</b>".format(text)
+            return text
+
+        def show_part(point_data, singular, plural, max_shown, vars):
+            if not vars:
+                return ""
+            cols = [format_val(var, point_data)
+                    for var in vars[:max_shown + 2]
+                    if vars == domain.class_vars
+                    or var not in (self.shown_x, self.shown_y)][:max_shown]
+            n_vars = len(vars)
+            if n_vars > max_shown:
+                cols[-1] = "... and {} others".format(n_vars - max_shown + 1)
+            return \
+                "<b>{}</b>:<br/>".format(singular if n_vars < 2 else plural) \
+                + "<br/>".join(cols)
+
+        def point_data(p):
+            point_data = self.data[p.data()]
+            text = "<br/>".join(
+                format_val(var, point_data, bold=self.tooltip_shows_all)
+                for var in (self.shown_x, self.shown_y))
+            if self.tooltip_shows_all:
+                text += "<br/><br/>" + \
+                        "<br/>".join(show_part(point_data, *columns)
```

Comment:
I suggest joining with "" here and removing one <br/> in the above line. And instead adding the <br/> at the beginning of line 1199.

Currently, if show_part returns an empty string it is still joined with the rest using <br/>, producing superfluous blank lines.